erpnext/manufacturing/doctype/production_order/production_order.js

```diff
@@ -235,9 +248,15 @@ $.extend(cur_frm.cscript, {
 
 	make_se: function(purpose) {
 		var me = this;
-		var max = (purpose === "Manufacture") ?
-			flt(this.frm.doc.material_transferred_for_manufacturing) - flt(this.frm.doc.produced_qty) :
-			flt(this.frm.doc.qty) - flt(this.frm.doc.material_transferred_for_manufacturing);
+		if(!this.frm.doc.skip_transfer){
+			var max = (purpose === "Manufacture") ?
+				flt(this.frm.doc.material_transferred_for_manufacturing) - flt(this.frm.doc.produced_qty) :
+				flt(this.frm.doc.qty) - flt(this.frm.doc.material_transferred_for_manufacturing);
+		} else if(this.frm.doc.skip_transfer){
```

Comment:
else {
     var max = flt(this.frm.doc.qty) - flt(this.frm.doc.produced_qty);
}